src/test/java/training/AppTest.java

```diff
@@ -1,14 +1,77 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
```

Comment:
Revert all the changes in this file there should not be any changes in apptest file.